projects/python-client/src/datapane/client/api/ipython_utils.py

```diff
@@ -19,10 +27,201 @@ def block_to_iframe(block: BaseElement) -> str:
     Returns:
         HTML string for the block
     """
-    # importing within function scope to avoid circular dependency
     from .report.core import App
 
     app = App(block)
     block_html_string = app.stringify(template_name="ipython_template.html")
 
     return block_html_string
+
+
+def read_notebook_json(path: Path) -> dict:
+    """Read a notebook IPYNB file
+
+    Args:
```

Comment:
I think we can drop `Args` and `Returns` from docstrings if using type annotations